src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/SortSamSpark.java

```diff
@@ -18,6 +19,44 @@
 import java.util.Collections;
 import java.util.List;
 
+
+/**
+ * SortSam on Spark (works on SAM/BAM/CRAM)
+ *
+ * <h4>Overview</h4>
+ * <p>A <a href='https://software.broadinstitute.org/gatk/blog?id=23420'>Spark</a> implementation of <a href='https://software.broadinstitute.org/gatk/documentation/tooldocs/current/picard_sam_SortSam.php'>Picard SortSam</a>. The Spark version can run in parallel on multiple cores on a local machine or multiple machines on a Spark cluster while still matching the output of the single-core Picard version. See <a href="https://software.broadinstitute.org/gatk/blog?id=23420">Blog#23420</a> for performance benchmarks.</p>
+ *
+ * <p>The tool sorts reads by coordinate order by default or alternatively by read name, the QNAME field, if asked with the '-SO queryname' option. The contig ordering in the reference dictionary defines coordinate order, and the tool uses the sequence dictionary represented by the @SQ header lines or that of the optionally provided reference to sort reads by the RNAME field. For those reads mapping to a contig, coordinate sorting further orders reads by the POS field of the SAM record, which contains the leftmost mapping position.</p>
+ *
+ *  <p>Queryname-sorted alignments are grouped first by readname and then are deterministically ordered among equal readnames by read flags including orientation, secondary, and supplemntary record flags (See {@link htsjdk.samtools.SAMRecordQueryNameComparator#compare(SAMRecord, SAMRecord)}} for details). For paired-end reads, reads in the pair share the same queryname. Because aligners can generate secondary and supplementary alignments, queryname groups can consists of, e.g. more than two records for a paired-end pair.</p>
+ *
+ * <h4>Usage examples</h4>
+ * Coordinate-sort aligned reads using all cores available locally
+ * <pre>
+ * gatk SortSamSpark \<br />
```

Comment:
As I mention below, here and elsewhere:
Please remove the `<br>` elements as they are unnecessary for gatkdocs and actually add a weird additional line between command lines.

Right now, this is what the doc looks like:
![screenshot 2019-03-06 14 05 00](https://user-images.githubusercontent.com/11543866/53906411-dc3eb280-4018-11e9-911d-0ac4efb6d57f.png)

We want to tighten up the line spacing and removing the breaks will do that for us.